sql/src/main/java/io/crate/execution/ddl/views/TransportCreateViewAction.java

```diff
@@ -105,11 +106,12 @@ protected CreateViewResponse newResponse(boolean acknowledged) {
         }
     }
 
-    private boolean conflictsWithTable(TableIdent viewName, MetaData indexMetaData) {
-        return indexMetaData.hasIndex(viewName.indexName());
+    private static boolean conflictsWithTable(TableIdent viewName, MetaData indexMetaData) {
```

Comment:
I guess we cannot have conflict with the name of a specific partition of a table, right?

Reply:
no, they're encoded using dot notation which is invalid in a relation name.